### ISSUE TITLE
dependency problems

### ISSUE THREAD

**Issue by logicminds** [neillturner/kitchen-puppet#188]
This code does not work like you think it does.  The end user does not execute this gemspec directly and this conditional logic is only run at build gem time.  So if you built the gem with ruby 2.3, everyone will be required to install test kitchen 1.17.  In my case I am running ruby 2.1 so I cannot install kitchen > 1.16 and therefore cannot install kitchen-puppet 3.1.2.

```
if RUBY_VERSION < '2.3'
    s.add_dependency 'test-kitchen', '<= 1.16.0', '>= 1.4'
  else
    s.add_dependency 'test-kitchen', '>= 1.17.0'
  end
  if RUBY_VERSION >= '2.0'
    s.add_dependency 'net-ssh', '>= 3'
  else
    s.add_dependency 'net-ssh', '~> 2.9'
  end
```

**Comment by neillturner** [comment]
ok thanks for that.  let me look into that

**Comment by neillturner** [comment]
closed. fixed in 3.1.3
